CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
```

Comment:
@AdrianSosic regarding your comment on "sustainability": Unfortunately, I do not see a different solution at the moment - I guess for the generics fix, we just need to wait until they fix that on the sphinx side. The only possible solution I found for the aliases was via using `TypeVar` which however only seems to work for Python 3.9+ and also caused some errors in the examples when testing this. I'd thus propose to rather have a quick+dirty solution then none at all. I'll also try to document this, such that we later still know what we did and why

Reply:
OK, we simply go with whatever you recommend here, hence I'll close. For the future: the `TypeVar` approach (whatever that looks like) could still be an options, since doc building happens on the highest python version anyway, and also code-wise, we are about to drop 3.8